manifests/paddock/overlays/smaug/image_stream_tag.yaml

```diff
@@ -7,7 +7,8 @@ spec:
   - from:
       kind: DockerImage
       name: ghcr.io/b4mad/paddock:latest
-    importPolicy: {}
+    importPolicy:
+      scheduled: true
     name: latest
```

Comment:
If the tag is latest, OpenShift Container Platform defaults imagePullPolicy to Always, I think we can omit pulling from source?!

https://docs.openshift.com/container-platform/3.11/dev_guide/managing_images.html#image-pull-policy